core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -83,6 +83,11 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EVENT_LOG_PERMISSION =
+    ConfigBuilder("spark.eventLog.permission")
+      .stringConf
```

Comment:
Better to add `doc` here and fill in the detailed illustration of the usage and default value.